### PR TITLE
Feature: cli debug and other commands introduced

### PR DESCRIPTION
**Pull request by fredriclam**
😎

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,164 +1,524 @@
...
+            reaction: "monkaS:506646811688173579"
+          }, (err) => {
+            if (err) {console.log(err); logger.error(`Reaction error: ${err.message}`)}
+          });
+          break;
+        case 'maximback':
+          interruptTimer(bot);
+          break;
+        case 'db': // Interactive debugger
+          if (flags.off && bot.isInteractive){
```
Comment (jackymayo) [inline app.js:216]: Why are there flags for interactive mode tho?
Shouldn't it just be !db?
Then you can toggle with 
```
if (bot.isInteractive) {
         ....
} else {
    ...
}
```

Comment (fredriclam) [reply app.js:216]: Hmmm good point, sounds like a good change.

Wanted to flex the flags parsing :)

File: app.js [hunk 2]
```diff
@@ -1,164 +1,524 @@
...
-      }
-      else if (msg.includes(bot.id)){
+    else if (bot.isInteractive){ //Not !command
+      // Handle interactive commands
+      logger.debug(`Interactive execution: ${message}`)
+      try{
         bot.sendMessage({
-            to: channelID,
-            message: "Hi! I'm a bot that simulates @mnhn329. Switch who I simulate with !learn <mention>!"
+          to: channelID,
```
Comment (jackymayo) [inline app.js:266]: Would be useful to have:

``` 
function isObject(val) {
    return val instanceof Object; 
}
let variable = eval(message)
if (isObject(variable)){
    // output JSON.stringify(variable)
}
else 
   // output variable
```

File: app.js [hunk 3]
```diff
@@ -1,164 +1,524 @@
...
+        line => hasTypeInLine("info", line));
+      logName = "Info logs: ";
+    }
+    else if (flags["debug"]){
+      ingestedLines = ingestedLines.filter(
+        line => hasTypeInLine("debug", line));
+      logName = "Debug (only) logs: ";
+    }
+    // Build log string
+    if (ingestedLines.length == 0)
```
Comment (jackymayo) [inline app.js:435]: Yay or nay? 

```
  let logString = "No logs found";
  if (ingestedLines.length != 0){
          logString = '```JSON\n' + ingestedLines.slice(-bot.dumpLength)
                              .join('\n```\n```JSON\n') + '```';
          logString = `${logName} (last ` + bot.dumpLength + " lines):\n" + logString;
 }
```

File: app.js (hunk 2 above)
Comment (fredriclam) [reply app.js:266]: 👍

File: app.js (hunk 3 above)
Comment (fredriclam) [reply app.js:435]: Slick

**Pull request merged**